src/main/kotlin/com/cosmotech/api/rbac/model/RbacAccessControl.kt

```diff
@@ -2,6 +2,6 @@
 // Licensed under the MIT license.
 package com.cosmotech.api.rbac.model
 
-import com.cosmotech.api.rbac.ROLE_NONE
+import com.cosmotech.api.rbac.ROLE_VIEWER
 
-open class RbacAccessControl(open var id: String = "emptyId", open var role: String = ROLE_NONE)
+open class RbacAccessControl(open var id: String = "emptyId", open var role: String = ROLE_VIEWER)
```

Comment:
I think that you can remove the default value.
`role` should be mandatory (like `id`) maybe the domain specific definition (on openapi side) and mappers should be adapt